.github/workflows/ci.yml

```diff
@@ -923,3 +923,43 @@ jobs:
     - name: run ./ci/docfx.cmd
       shell: cmd
       run: ./ci/docfx.cmd
+
+  w3c_trace_context_compliance_v1:
+    name: W3C Distributed Tracing Validation V1
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout open-telemetry/opentelemetry-cpp
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+      - name: setup
+        env:
+          CC: /usr/bin/gcc-10
+          CXX: /usr/bin/g++-10
+          PROTOBUF_VERSION: 21.12
+        run: |
+          sudo -E ./ci/setup_googletest.sh
+          sudo -E ./ci/setup_ci_environment.sh
+      - name: run w3c trace-context test server (background)
+        env:
+          CXX_STANDARD: '14'
+        run: |
+          ./ci/do_ci.sh cmake.w3c.trace-context.build-server
+          cd $HOME/build/ext/test/w3c_tracecontext_test
+          ./w3c_tracecontext_test &
```

Comment:
Is the tailing `&` for running the test in background? If so, is it needed?

Reply:
It is starting the test web server, so required in background. But in general good to invoke such services under `nohup` so that it doesn't corrupt the CI terminal output.

Reply:
Yes, I also just read that in the w3c_tracecontext_test, probably better to imply that is is test server in the file name:)

Reply:
@ThomsonTan, just to confirm you're suggesting to change the name of the build output file `w3c_tracecontext_test` &rarr; `w3_tracecontext_test_server` ? 

I could do that, but I feel that line 943 conveys that this is a test server running in background ?

Reply:
Yes, renaming `w3c_tracecontext_test` -> `w3c_tracecontext_test_server` will be great, and we can also do this in a separate PR.

Reply:
This is being addressed in PR #3132.